templates/html-header.twig

```diff
@@ -56,14 +56,28 @@
           }
 
           if ( google_tag_value ) {
-			window.dataLayer = window.dataLayer || [];
+			var dataLayer = window.dataLayer = window.dataLayer || [];
```

Comment:
is this intended? I mean can't it be `var dataLayer = window.dataLayer || [];`

Reply:
Yes it is intended.(https://www.bounteous.com/insights/2016/03/21/gtm-data-layer-best-practices/#proper-datalayer-instantiation)
Julia also recommended to use it as `var dataLayer = window.dataLayer = window.dataLayer || [];`